nuget/Microsoft.Windows.CppWinRT.targets

```diff
@@ -248,11 +248,12 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
     <!--Get direct WinMD references (including Nuget packages) for projections, IDL processing, and AppX packaging-->
     <Target Name="GetCppWinRTDirectWinMDReferences"
-            DependsOnTargets="ResolveAssemblyReferences;$(GetCppWinRTDirectWinMDReferencesDependsOn)"
+            DependsOnTargets="ExpandSDKReferences;ResolveAssemblyReferences;$(GetCppWinRTDirectWinMDReferencesDependsOn)"
```

Comment:
This is unnecessary as ResolveAssemblyReferences already depends on it, but also doesn't hurt to have it here.